src/constants.js

```diff
@@ -106,6 +106,11 @@ export const MAX_OUTPUT_VALUE_32 = 2 ** 31 - 1;
  */
 export const MAX_OUTPUT_VALUE = 2 ** 43;
 
+/**
+ * Max number of inputs in a transaction
+ */
+export const MAX_INPUTS = 255;
```

Comment:
No need to add this. We already have this value that we get from the server. You can access with `transaction.getMaxInputsConstant()`